pegasus/sites/code.org/views/page_header.haml

```diff
@@ -16,10 +16,6 @@
 
     #clear{:style=>'clear:both'}
 
-  #loc.desktop-feature
```

Comment:
If you look at ro.code.org, we used to have a gray header where we showed the # lines of code just under the main navigation. This was only used on the code.org homepage (not on csedweek.org and I think even before hoc.com). We redesigned the code.org homepage, so it's not used there anymore. And the chapter partner sites don't use it because they have their own localized text.